Dockerfile

```diff
@@ -1,38 +1,28 @@
 FROM node:10.13.0
 
-# Set up deploy user and working directory
-RUN adduser --disabled-password --gecos '' deploy
-RUN mkdir -p /app
+WORKDIR /app
 
-# Set up dumb-init
-ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 /usr/local/bin/dumb-init
-RUN chown deploy:deploy /usr/local/bin/dumb-init
-RUN chmod +x /usr/local/bin/dumb-init
+# Set up deploy user and working directory
+RUN apt-get update && apt-get install \
+  dumb-init \
+  libgpm2 \
+  libslang2 \
+  mc \
```

Comment:
`mc` and its dependencies are added here, is there a specific preference to import in the way below?

Reply:
Doh!  Okay then, no that's fine (better even) that it's installed via a package :)